src/KuduClient.ts

```diff
@@ -5,9 +5,11 @@
 
 import * as kuduApi from 'kudu-api';
 import * as request from 'request';
+import { IncomingMessage } from 'http';
 
 export type kuduFile = { mime: string, name: string, path: string };
 export type webJob = { name: string, Message: string };
+export type kuduIncomingMessage = IncomingMessage & { body: string };
 
 /*
```

Comment:
Please delete the file-related methods to prevent new code from leveraging it

Reply:
When you say file-related methods, do you mean the type definitions that I created?